test/benchmark/run_performance_tracker.sh

```diff
@@ -3,4 +3,19 @@
 # change to script directory
 cd "${0%/*}" || exit
 
+# verify if the benchmark/sift directory exists and if it doesn't create it
+if [ ! -d "./sift" ]; then
+  mkdir ./sift
+fi
```

Comment:
Could just `mkdir -p ./sift`, no?

Reply:
Sure, that would also work.
Thanks for the review, I'll adapt it.